### ISSUE TITLE
Provide product archive downloads (without Java)

### ISSUE THREAD

**Issue by peterkir** [amitjoy/osgifx#762]
As a developer inside company networks I want to download the product archive and be able to extract it and run it with my local available system java instance.
Archive download it feasible for most developers. Executables are filtered by firewall/proxy settings.

**Comment by amitjoy** [comment]
Yeah, you are absolutely right. I would need to make the executable jar available to the release pages. The executables are built for ease of use, but executable jar is a better choice, in which case, you need to have the required JDK with JavaFX manually installed to your system.

**Comment by peterkir** [comment]
![enterprise_install](https://github.com/user-attachments/assets/829de1b0-7335-435b-ba1b-919d0de8cca6)
Archives would allow to download and use it inside corporate networks.
On windows I get above error, when downloading and using the Installer executable.
So a zip archive (with or without java) would be a nice workaround.

**Comment by amitjoy** [comment]
Version 2.4.5 is now available for download as a standalone JAR file. This allows for a direct launch without the need for a full product installation. For detailed instructions, please visit https://www.osgifx.com/headless-launch for further info.
